minitests/iostandard/features/analyze.py

```diff
@@ -0,0 +1,168 @@
+import os
+import argparse
+import json
+import random
+
+from collections import defaultdict
+
+from prjxray import util
+import fasm
+
+# =============================================================================
+
+
+def load_iob_segbits():
```

Comment:
Why not use the prjxray python library?  Then you don't have to assume anything w.r.t. `LIOB33`/`RIOB33`.  You also have to deal with the fact that `_SING` tiles have a different offset, which would be handled if you used the library decode functions?

Reply:
But I'm not really decoding anything here. I just need a list of fasm features of an IOB site.